src/screens/ContactScreen/index.js

```diff
@@ -82,7 +139,7 @@ const ContactScreen = ({navigation}) => {
             switch (type) {
               case VIEW_TYPE_DATA:
                 dim.width = width;
-                dim.height = 72;
+                dim.height = 64;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The height of the `VIEW_TYPE_DATA` has been reduced from 72 to 64. Ensure that this change doesn't affect the layout or visibility of the data.
```diff
- dim.height = 72;
+ dim.height = 64;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->